components/chef-run/lib/chef-run/cli.rb

```diff
@@ -188,16 +91,16 @@ def handle_run_error(e)
 
     def perform_run
       parse_options(@argv)
-      if @argv.empty? || config[:help]
+      if @argv.empty? || parsed_options[:help]
```

Comment:
I really hate that mixlib-cli exposes the parsed options in a `config` method, so I wrote a wrapper and called it `parsed_options` - definitely open to a better name if someone thinks of one